skycatalogs/objects/snana_object.py

```diff
@@ -43,7 +48,9 @@ def get_observer_sed_component(self, component, mjd=None):
             sed = self._apply_component_extinction(sed)
         return sed
 
-    def get_LSST_flux(self, band, sed=None, cache=True, mjd=None):
+    def get_LSST_flux(self, band, sed=None, cache=False, mjd=None):
+        # There is usually no reason to cache flux for SNe, in fact it could
+        # cause problems
```

Comment:
What are the cases where it can cause problems?  I think the implications of this sort of comment need to be spelled out and not left unstated like this.